galette/lib/Galette/Entity/Adherent.php

```diff
@@ -566,6 +566,7 @@ private function checkDues(): void
                         $this->_due_status = Contribution::STATUS_UPTODATE;
                     }
                 // Expired
+                    // Expired
```

Comment:
Please avoid IDE automatic changes
```suggestion
                // Expired
```